include/api.php

```diff
@@ -303,9 +323,10 @@ function pll_languages_list( $args = array() ) {
  *
  * @api
  * @since 1.5
+ * @since 3.4 $lang accepts PLL_Language or string.
  *
- * @param int    $id   Post id.
- * @param string $lang Language code.
+ * @param int                 $id   Post id.
+ * @param PLL_Language|string $lang Language (object or slug).
  * @return void
  */
 function pll_set_post_language( $id, $lang ) {
```

Comment:
Yeah, why not, this shouldn't break anything and since we have the data at hand...
Done in [fda227d](https://github.com/polylang/polylang/pull/1149/commits/fda227d7895656589e0c94c73d20db74e2395790).